comparator/src/main/java/org/mobilitydata/gtfsvalidator/outputcomparator/util/NoticeAggregate.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.outputcomparator.util;
+
+import java.util.List;
+import java.util.Map;
+import org.mobilitydata.gtfsvalidator.notice.SeverityLevel;
+
+/** Used to deserialize a validation report */
```

Comment:
Please add more information: what is kept in this class?

Please also put a full stop at the end of each sentence in your comments.